doc/modules/cross_validation.rst

```diff
@@ -467,44 +378,181 @@ Here is a usage example::
 validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
+Cross-validation iterators with stratification based on class labels.
+=====================================================================
+
+In some datasets the number of samples per class varies based on the class
+label. They are referred to commonly as datasets with imbalance in classes.
+In such cases, stratified sampling may be done to pick the samples for
+cross-validation. This avoids the class imbalance from affecting the
+cross-validation results.
 
-Label-Shuffle-Split
+The :class:`StratifiedKFold` and :class:`StratifiedShuffleSplit` do
+stratified cross-validation to ensure that the proportion of samples from each
```

Comment:
that's not right.... ``do stratified cross-validation, that is they ensure that each training and test set have the same class frequencies as the overall data (up to rounding)``.